packages/StoryCard/StoryCard.jsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { News, Headset, Play, Clipboard } from '@tds/core-decorative-icon'
+import {
+  StyledCard,
+  StyledTextBox,
+  StyledInfoBox,
+  StyledType,
+  StyledDate,
+  StyledHeadingBox,
+  StyledHeading,
+  StyledDescription,
+  StyledIconTypeBox,
+  StyledLink,
+  StyledIconBox,
+  StyledImageContainer,
+  StyledImage,
+} from './style'
+
+const StoryCard = ({ storyType, date, title, description, imgUrl, slug }) => {
+  const selectIcon = type => {
+    const normalizedType = type.toLowerCase()
```

Comment:
You'll need a check here if no storyType is undefined, toLowerCase crashes the component if no prop is passed